numba_cuda/numba/cuda/extending.py

```diff
@@ -352,6 +360,9 @@ def impl(cls, nitems):
         def foo(n):
             return types.Array.make(n)
     """
+
+    kwargs["target"] = "cuda"
```

Comment:
Is this what we intended here?
```suggestion
    kwargs["target"] = target
```

Reply:
Whoops!